<p>2018 Ebola virus disease outbreak in Équateur Province, Democratic Republic of the Congo: a retrospective genomic characterisation
Mbala-Kingebeni P, Pratt CB, Wiley MR, Diagne MM, Makiala-Mandanda S, Aziza A, Di Paola N, Chitty JA, Diop M, Ayouba A, Vidal N, Faye O, Faye O, Karhemere S, Aruna A, Nsio J, Mulangu F, Mukadi D, Mukadi P, Kombe J, Mulumba A, Duraffour S, Likofata J, Pukuta E, Caviness K, Bartlett ML, Gonzalez J, Minogue T, Sozhamannan S, Gross SM, Schroth GP, Kuhn JH, Donaldson EF, Delaporte E, Sanchez-Lockhart M, Peeters M, Muyembe-Tamfum JJ, Alpha Sall A, Palacios G, Ahuka-Mundeke S</p>

<p>Abstract
Background: The 2018 Ebola virus disease (EVD) outbreak in Équateur Province, Democratic Republic of the Congo, began on May 8, and was declared over on July 24; it resulted in 54 documented cases and 33 deaths. We did a retrospective genomic characterisation of the outbreak and assessed potential therapeutic agents and vaccine (medical countermeasures).</p>

<p>Methods: We used target-enrichment sequencing to produce Ebola virus genomes from samples obtained in the 2018 Équateur Province outbreak. Combining these genomes with genomes associated with known outbreaks from GenBank, we constructed a maximum-likelihood phylogenetic tree. In-silico analyses were used to assess potential mismatches between the outbreak strain and the probes and primers of diagnostic assays and the antigenic sites of the experimental rVSVΔG-ZEBOV-GP vaccine and therapeutics. An in-vitro flow cytometry assay was used to assess the binding capability of the individual components of the monoclonal antibody cocktail ZMapp.</p>

<p>Findings: A targeted sequencing approach produced 16 near-complete genomes. Phylogenetic analysis of these genomes and 1011 genomes from GenBank revealed a distinct cluster, confirming a new Ebola virus variant, for which we propose the name “Tumba”. This new variant appears to have evolved at a slower rate than other Ebola virus variants (0·69 × 10-3 substitutions per site per year with “Tumba” vs 1·06 × 10-3 substitutions per site per year without “Tumba”). We found few sequence mismatches in the assessed assay target regions and antigenic sites. We identified nine amino acid changes in the Ebola virus surface glycoprotein, of which one resulted in reduced binding of the 13C6 antibody within the ZMapp cocktail.</p>

<p>Interpretation: Retrospectively, we show the feasibility of using genomics to rapidly characterise a new Ebola virus variant within the timeframe of an outbreak. Phylogenetic analysis provides further indications that these variants are evolving at differing rates. Rapid in-silico analyses can direct in-vitro experiments to quickly assess medical countermeasures.</p>
